README.md-review

```diff
@@ -0,0 +1,107 @@
+# Operator for IBM block storage CSI driver
+The Container Storage Interface (CSI) Driver for IBM block storage systems enables container orchestrators such as Kubernetes to manage the life-cycle of persistent storage.
+
+This is the official operator to deploy and manage IBM block storage CSI driver.
+
+Supported container platforms:
+  - Openshift v4.1
+  - Kubernetes v1.13
+
+Supported operating systems:
+  - RHEL 7.x (x86 architecture)
+  - RHCOS 4.1 (x86 architecture)
+
+## Prerequisite
+Please go to section [`Prerequisites for Driver Installation`](https://github.com/IBM/ibm-block-csi-driver#prerequisites-for-driver-installation) for details.
+
+## Installation
+
+### Install the operator
+1. Download the manifest from GitHub.
+```bash
+curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/develop/deploy/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
+```
+2. Update the image fields in the ibm-block-csi-operator.yaml if the default values are not fit for you.
+3. Install the operator
+
+<!-- $ kubectl apply -f csi_driver.yaml  (download and install csi_driver.yaml only if you are using Kubernetes v.1.14+) -->
+```bash
+$ kubectl apply -f ibm-block-csi-operator.yaml
+```
+
+### Verify operator is running:
+
+```bash
+$ kubectl get pod -l app.kubernetes.io/name=ibm-block-csi-operator -n kube-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+ibm-block-csi-operator-5bb7996b86-xntss 2/2     Running   0          10m
+```
+
+### Create an IBMBlockCSI custom resource
+1. Create an IBMBlockCSI yaml file (ibc.yaml) as following and update the repository and tag if the default values are not fit for you:
```

Comment:
1. Create an IBMBlockCSI yaml file (ibc.yaml). If required, update the repository and tag values.